client/packages/programs/src/api/hooks/document/useProgramEnrolments.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  ProgramEnrolmentSortFieldInput,
+  useMutation,
+  useQuery,
+} from '@openmsupply-client/common';
+import { ProgramEnrolmentListParams } from '../../api';
+import { ProgramEnrolmentRowFragment } from '../../operations.generated';
+import { useProgramEnrolmentApi } from '../utils/useProgramEnrolmentApi';
+
+export type ProgramEnrolmentRowFragmentWithId = {
+  id: string;
+} & ProgramEnrolmentRowFragment;
+
+export const useProgramEnrolmentsPromise = () => {
+  const api = useProgramEnrolmentApi();
+
+  return {
```

Comment:
no need to make a new object

Reply:
Thanks fixed + some other occurrences.

Pretty sure it's a copy and past issue (stocktake and invoice use it similarly...)